To: vim_dev@googlegroups.com
Subject: Patch 7.3.131
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.131
Problem:    Including errno.h too often.
Solution:   Don't include errno.h in Unix header file.
Files:	    src/os_unix.h


*** ../vim-7.3.130/src/os_unix.h	2010-08-15 21:57:28.000000000 +0200
--- src/os_unix.h	2010-12-17 20:05:34.000000000 +0100
***************
*** 184,193 ****
  
  #define BASENAMELEN	(MAXNAMLEN - 5)
  
- #ifdef HAVE_ERRNO_H
- # include <errno.h>
- #endif
- 
  #ifdef HAVE_PWD_H
  # include <pwd.h>
  #endif
--- 184,189 ----
*** ../vim-7.3.130/src/version.c	2011-02-25 15:15:43.000000000 +0100
--- src/version.c	2011-02-25 15:16:49.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     131,
  /**/

-- 
Overflow on /dev/null, please empty the bit bucket.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
